decidim-accountability/app/jobs/decidim/accountability/admin/import_projects_job.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Accountability
+    module Admin
+      class ImportProjectsJob < ApplicationJob
+        queue_as :default
+
+        def perform(projects, current_component, current_user)
```

Comment:
The job itself is and should be unaware of the context it is running in, so could we just rename these `component` and `user` instead? Please also note that you have to change the variable names accordingly below as well.

```suggestion
        def perform(projects, component, user)
```